accepted/00000-auto-generate-bootstrap-repositories.md

```diff
@@ -0,0 +1,71 @@
+- Feature Name: Autogenerate Bootstrap Repositories
+- Start Date: 2020-02-19
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Automatically generate bootstrap repositories
+
+# Motivation
+[motivation]: #motivation
+
+To be able to register a system to Uyuni Server you need to have the software installed which provide the required tooling.
+
+A bootstrap repository provide the software for the supported operating systems with its dependencies. The content for the
+bootstrap repositories come from the normal channels which must be synchronized and updated on regular base.
+
+Users very often do not know this, or forget to create them when they synchronize a new product. Also the bootstrap repository
+is only updated when the user execute the generator script again.
+
+This RFC should describe how to change the current tools to automatically 
+execute the generation of bootstrap repositories after
+a new product was added and after a channel changed when a maintenance update was synced to Uyuni Server.
```

Comment:
Do we also need to create/update the bootstrap repository when the product itself is synchronized, e. g. new or updated packages in the openSUSE Leap Updates repository potentially every day?

Reply:
yes, exactly. I expect they will be re-generated daily.